entity-types/infra-mysqlnode/golden_metrics.yml

```diff
@@ -26,6 +29,10 @@ slowQueriesPerMinute:
       from: MysqlSample
       eventId: entityGuid
       eventName: entityName
+    opentelemetry:
+      select: average(`mysql.query.slow.count`)
```

Comment:
I have the same doubt, if the metric before was `slowQueriesPerSecond` we could expect a non-monotonic number. 
In this case, even if we compute the average, isn't `mysql.query.slow.count` always increasing? I fear we are changing the meaning of this golden metric (unless I am missing something about how this average is computed).

Reply:
I have commented out these golden metrics for now, as they are not compatible with existing golden metrics